pkg/api/auth/token/user.go

```diff
@@ -0,0 +1,30 @@
+package token
+
+type User interface {
+	// Name
+	GetName() string
```

Comment:
No need to put explicit `Get` here, the method here behaves like an attribute to User, it's more go style

Reply:
If the UID field is defined, the UID method cannot be defined, https://github.com/kubernetes/apiserver/blob/master/pkg/authentication/user/user.go#L53-L63